devops/tmpnightly.yml

```diff
@@ -0,0 +1,80 @@
+trigger: none
+pr: none
+
+schedules:
+  - cron: "30 5 * * *" # Time is UTC
+    displayName: Nightly Code Coverage Build
+    branches:
+      include:
+        - master
+    always: true
+
+jobs:
+- job: Linux
+  pool:
+    vmImage: 'ubuntu-latest'
+  steps:
+  - script: echo hello from Linux
+- job: macOS
+  pool:
+    vmImage: 'macOS-latest'
+  steps:
+  - script: echo hello from macOS
```

Comment:
What's going on with mac and windows here?

Reply:
they were placeholders but I'll pull them out and rerun to make sure all is fine.